README.md

```diff
@@ -51,6 +51,7 @@ Read the [documentation](https://unocss.dev/) for more details.
 
 - [Vite](https://unocss.dev/integrations/vite)
 - [Nuxt](https://unocss.dev/integrations/nuxt)
+- [Next.js](https://unocss.dev/integrations/next)
```

Comment:
The reason we don't put it here is that unlike the others, we don't provide direct integrations with Next.js. It's more like a guide than an integration

Reply:
makes sense, well would there be a place that fits a guide for next?

Reply:
I think we could have a `Guide` section under https://unocss.dev/integrations/#frameworks-tools